testsupport/tiers/tier_setup.go

```diff
@@ -156,12 +149,25 @@ func duplicateTierTemplate(hostAwait *HostAwaitility, namespace, tierName, origT
 	return newTierTemplate.Name, nil
 }
 
-func MoveUserToTier(t *testing.T, hostAwait *HostAwaitility, username, tierName string) {
-	t.Logf("moving user '%s' to tier '%s'", username, tierName)
-	changeTierRequest := NewChangeTierRequest(hostAwait.Namespace, username, tierName)
-	err := hostAwait.CreateWithCleanup(context.TODO(), changeTierRequest)
+func MoveSpaceToTier(t *testing.T, hostAwait *HostAwaitility, spacename, tierName string) {
+	t.Logf("moving space '%s' to space tier '%s'", spacename, tierName)
+	space, err := hostAwait.WaitForSpace(spacename)
+	require.NoError(t, err)
+
+	space.Spec.TierName = tierName
+
+	err = hostAwait.Client.Update(context.TODO(), space)
+	require.NoError(t, err)
```

Comment:
I have updated https://github.com/codeready-toolchain/host-operator/pull/637 so that the space controller will remove the old tier hash label as part of the space update. Also, the deactivation controller now resets the Deactivating state so that when a user is promoted they will still receive the Deactivating notification at the correct time so this fixes the e2e test failure for this PR. This also means we don't need to use ChangeTierRequest for tier updates anymore, can promote MURs and Spaces individually.